history.php

```diff
@@ -0,0 +1,57 @@
+<?php
+session_start();
+require 'db_connection.php';
+// --------------------------------------------------
+// ログインユーザーを取得する
+// --------------------------------------------------
+$user_name = $_SESSION['name'];
+$user_id = $_SESSION['id'];
+// --------------------------------------------------
+// 履歴を取得
+// --------------------------------------------------
+if (isset($db)) {
+    $sql = 'select * from histories';
```

Comment:
採点結果はログインユーザーのもののみ表示で良いので
```suggestion
    $sql = 'select * from histories where user_id = :user_id';
```
最初からuser_idが一致したものだけを取得でも良いかと思います。